wire-ios/Wire-iOS/Sources/UserInterface/Settings/Backup/Adapters/MessageStoreAdapter/MessageContent+initWithGenericMessageContent.swift

```diff
@@ -105,12 +105,12 @@ extension MessageContent {
             name: .none,
             otrKey: imageAsset.otrKey,
             sha256: imageAsset.sha256,
-            assetID: "",
+            assetID: "", // TODO: what value to set? is empty string ok?
```

Comment:
Not sure about this.
Do we have an assetID and I missed it?
Is it required?
Maybe we cannot support the legacy image type at all.